custom_components/dahua/__init__.py

```diff
@@ -102,11 +102,7 @@ def __init__(self, hass: HomeAssistant, events: list, address: str, port: int, r
                  password: str, name: str, channel: int) -> None:
         """Initialize the coordinator."""
         # Self signed certs are used over HTTPS so we'll disable SSL verification
-        ssl_context = ssl.create_default_context()
-        ssl_context.set_ciphers("DEFAULT")
-        ssl_context.check_hostname = False
-        ssl_context.verify_mode = ssl.CERT_NONE
-        connector = TCPConnector(enable_cleanup_closed=True, ssl=ssl_context)
+        connector = TCPConnector(enable_cleanup_closed=True, ssl=False)
```

Comment:
I don't have any devices running ssl so I'm not able to test. I could merge and hope for the best?

Reply:
The original way is triggering hassio warning but works, the one presented here throws error as Dahua Cams, at least mine, are not capable of tlsv1.3 and it all ends in error
`  File "/usr/local/lib/python3.12/site-packages/aiohttp/connector.py", line 1084, in _wrap_create_connection
    raise ClientConnectorSSLError(req.connection_key, exc) from exc
aiohttp.client_exceptions.ClientConnectorSSLError: Cannot connect to host 192.168.68.242:443 ssl:default [[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] ssl/tls alert handshake failure (_ssl.c:1000)]
`

Reply:
Merged this https://github.com/rroller/dahua/pull/394

Reply:
Released in https://github.com/rroller/dahua/releases/tag/0.9.61

Can you please give it a try